gcloud/datastore/connection.py

```diff
@@ -149,7 +149,7 @@ def mutation(self):
                   (if one exists) or or a new mutation instance.
         """
         if self.transaction():
-            return self.transaction().mutation()
+            return self.transaction().mutation
```

Comment:
+1 for a review of the mocks, including removal / adjustment for minimalism.

-1 for centralizing them.  Quoting from my [Avoiding temptation](http://palladion.com/home/tseaver/obzervationz/2008/unit_testing_notes-20080724) rant:

  > -  Because they are not in the module which uses them, shared mock objects and fixtures makes impose a lookup burden on the reader.
  > -  Because they have to support APIs used by multiple clients, shared fixtures tend grow to grow APIs / data structures needed only by one client: in the degenerate case, become as complicated as the application they are supposed to stand in for!